internal/core/Cargo.toml

```diff
@@ -78,6 +78,7 @@ unicode-linebreak = { version = "0.1.2", optional = true }
 unicode-script = { version = "0.5.3", optional = true }
 integer-sqrt = { version = "0.1.5" }
 bytemuck = { workspace = true, optional = true, features = ["derive"] }
+smol_str = { workspace = true }
```

Comment:
i don't think we use smol_str in core and if we did we'd need to disable the std feature (that's why the CI is failing for the mcu build)

Reply:
fixed